google-cloud-pubsub/lib/google/cloud/pubsub/subscriber.rb

```diff
@@ -66,8 +109,15 @@ def start
           self
         end
 
+        ##
+        # Stops the subscriber.
```

Comment:
What happens after stop is called? Is there more to the story of calling `stop` on the streams?